parametric/test_span_sampling.py

```diff
@@ -442,3 +442,99 @@ def test_multi_rule_independent_rate_limiters_sss013(test_agent, test_library):
             pass
     span = find_span_in_traces(test_agent.wait_for_num_traces(1), Span(name="web.request"))
     assert span["metrics"].get(SINGLE_SPAN_SAMPLING_MAX_PER_SEC) == 1
+
+
+@pytest.mark.skip_library("golang", "Not implemented")
+@pytest.mark.skip_library("dotnet", "Not implemented")
+@pytest.mark.skip_library("nodejs", "Not implemented")
+@pytest.mark.skip_library("python", "Not implemented")
+@pytest.mark.parametrize(
+    "library_env",
+    [
+        {
+            "DD_SPAN_SAMPLING_RULES": json.dumps(
+                [{"service": "webserver", "name": "web.request", "sample_rate": 1.0, "max_per_second": 50}]
+            ),
+            "DD_TRACE_SAMPLING_RULES": json.dumps([{"sample_rate": 0}]),
```

Comment:
```suggestion
            "DD_TRACE_SAMPLE_RATE": 0,
```
Is there a reason to use `DD_TRACE_SAMPLNG_RULES` here instead? I think some tracers may not wildcard service when left out. Maybe best to stick with `DD_TRACE_SAMPLE_RATE` ?

Reply:
There is no specific reason for that, DD_TRACE_SAMPLE_RATE should be just fine for this test purposes